dom.bs

```diff
@@ -671,6 +672,11 @@ list of tuples, each of which consists of an <b>item</b> (an {{EventTarget}} obj
  <dd>Returns the object whose <a>event listener</a>'s <b>callback</b> is currently being
  invoked.
 
+ <dt><code><var>event</var> . {{Event/composedPath()}}</code>
+ <dd>Returns <var>event</var>'s path, which includes objects on which listeners will be invoked,
+ except for any <a>nodes</a> in <a>shadow trees</a> of which the <a for=/>shadow root</a>'s
```

Comment:
This sentence might not be accurate, however, I think it's okay because the normative behavior is exactly defined in the paragraph of the steps.